src/components/Library/LibraryForm/LibraryForm.tsx

```diff
@@ -13,30 +14,32 @@ export default function LibraryForm() {
     pages: '',
   };
 
+  const { t } = useTranslation();
+
   const formik = useFormik({
     initialValues,
     validationSchema: yup.object({
       name: yup
         .string()
-        .min(3, 'Min 3 symbols')
-        .max(50, 'Max 50 symbols')
-        .required('Required'),
+        .min(3, 'validation.min3')
+        .max(50, 'validation.max50')
```

Comment:
I think the name can have more than 50 symbols

Reply:
Ok